### DISCUSSION TITLE
Aggressive Fan Curves

### DISCUSSION THREAD

**Discussion by merrkry** [seerge/g-helper#982] [General]
I have a TUF FX507ZR which usually gets overheated while playing CPU-heavy large games (e.g. 2077 and Remnant 2).

By setting fan speed to 6000 at 80°C+, the problem seems to be settled. 

1. Will this significantly reduce fan life span?
2. In G-helper's graph the maximum value is 6400 while the curves peak at respectively 6000/6200. Is it safe to set to 6400?

**Comment by seerge** [comment]
1. No idea, probably not
2. Graphs Y-axis is indication only (real values behind that is a 0 - 100% values that are being set as curve point). By dragging dot to the top you just set 100% not RPM

**Reply by mikezila** [reply]
Would it be possible to change the UI to show 0-100% rather than bogus RPM values? I spent a while today looking at my current fan speed and trying to make sense of the curve. Right now g-helper will tell you the real fan RPM on the main window, but when you then open the curve editor the current temperature does not line up with the RPM value on the graph, which made me thing something was not working.

Having the curve editor show totally made up values instead of a simple 0-100% range is very confusing.

**Reply by seerge** [reply]
@mikezila i'm afraid it will raise even more questions then.

As for ASUS 0% - is OFF, while 1% - is immediately 1700-1800RPM for the fan (while 100% - is a some abstract maximum specific for each model / fan)

Keep in mind that fans are always controlled by BIOS (not by the app), and its up to BIOS how to threat even custom fan curve, it can have own algorithm / delays / hysteresis. So don't expect to see exact same RPM that you have set for specific temperature in a readings all the time :)

**Reply by mikezila** [reply]
What a mess the Asus bios is. Thanks for the explanation. It would be helpful if this information/context was in the app itself. perhaps a (i) icon someplace with help text explaining how the fan curve editor behaves and how the BIOS takes it as a suggestion instead of an order.
